ci/e2e.go

```diff
@@ -79,3 +79,20 @@ func postgres() *dagger.Service {
 		WithExposedPort(5432).
 		AsService()
 }
+
+const (
+	SvixJWTSingingSecret = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpYXQiOjE3MjI5NzYyNzMsImV4cCI6MjAzODMzNjI3MywibmJmIjoxNzIyOTc2MjczLCJpc3MiOiJzdml4LXNlcnZlciIsInN1YiI6Im9yZ18yM3JiOFlkR3FNVDBxSXpwZ0d3ZFhmSGlyTXUifQ.PomP6JWRI62W5N4GtNdJm2h635Q5F54eij0J3BU-_Ds"
+)
+
+func svix() *dagger.Service {
+	return dag.Container().
+		From(fmt.Sprintf("svix/svix-server")).
```

Comment:
Should we pin the version? (I usually like pinning)

(If not, `fmt.Sprintf` is not necessary)

Reply:
We should. I jsut forgot to add the version here. Will fix it.